types/src/chain_id.rs

```diff
@@ -21,8 +21,8 @@ pub enum NamedChain {
     MAINNET = 1,
     // The Libra chain below are non-reserved, non-production, and may change over time.  They are listed for convenience here.
     TESTNET,
-    DEVNET,
-    TESTING,
+    DEVNET = 5,
+    TESTING = 6,
```

Comment:
@davidiw : Why was there a breaking change to chain_id numbers? Can we use the previous numbers? This breaks AOS and partners whom we gave the chain id of `4` for `TESTING`